docs/userguide/settings.md

```diff
@@ -0,0 +1,213 @@
+# Settings
```

Comment:
@AVHopp, @Scienfitz, @kalama-ai 

This is just my proposal for the naming, but we can discuss if better approaches exist. To be decided:
* name of the module (currently: a public `settings.py`)
* name of the settings class (currently: `Settings`). I think other packages oftentimes use small letters because they put the user experience first, i.e. a user would e.g. expect to call `baybe.settings(...)` without having to worry that this is actually instantiating class behind the scenes
* name of the "global object" (currently: `active_settings`). 

Reasons for the current choice: all names are sufficiently different and rather explicit so that there is no confusion.

But happy to hear your input

Reply:
alright, then let me close. If anyone of the others would like to raise their voice, please reopen 🙃